tap_process_street/discover.py

```diff
@@ -1,22 +1,28 @@
+from singer import metadata
 from singer.catalog import Catalog, CatalogEntry, Schema
 
-from tap_process_street.schema import get_schemas, STREAMS
+from tap_process_street.constant import UPDATED_DATE_KEY, ID_KEY
+from tap_process_street.schema import get_schemas
 
 
 def discover():
-    schemas, schemas_metadata = get_schemas()
-
     streams = []
-    for schema_name, schema_dict in schemas.items():
+    for schema_name, schema_dict in get_schemas().items():
         schema = Schema.from_dict(schema_dict)
-        schema_meta = schemas_metadata[schema_name]
+        schema_meta = metadata.get_standard_metadata(
+            schema=schema_dict,
+            schema_name=schema_name,
+            key_properties=[ID_KEY],
+            valid_replication_keys=[UPDATED_DATE_KEY, ID_KEY],
+            replication_method='INCREMENTAL'
+        )
```

Comment:
I pulled this up so that the `schema.py` only deals with loading the raw schemas and this `discover.py` processes them and builds the metadata & catalog.